packages/core/src/awsService/appBuilder/wizards/serverlessLandWizard.ts

```diff
@@ -0,0 +1,290 @@
+/*!
```

Comment:
how much of this project has code related to serverlessLand? Does it make sense to invert the file structure and do:
`appBuilder/serverlessLand/wizards` ? or something like that? Might stop the serverless land code from having to live in a ton of different folders

Reply:
In total the project will have around 4-5 new files. We can create a new file structure or keep adding in the current structure. @valerena, what do you think?

Reply:
Thanks @valerena and @jpinkney-aws. I have added the files inside a new `serverlessLand` directory.